src/chigame/api/urls.py

```diff
@@ -3,17 +3,19 @@
 from . import views
 
 urlpatterns = [
-    # GAME API URLS
     path("games/", views.GameListView.as_view(), name="api-game-list"),
     path("games/<int:pk>/", views.GameDetailView.as_view(), name="api-game-detail"),
-    # LOBBY API URLS
+    path("tournaments/create/", views.TournamentCreateView.as_view(), name="create-tournament"),
+    path("tournaments/", views.TournamentListView.as_view(), name="tournament-list"),
+    path("register/", views.UserRegistrationView.as_view(), name="user-registration"),
```

Comment:
What is the point of this register endpoint?
Upon trying to access it it, it doesn't allow get. 
You don't mention it in your JWT walkthrough, nor have any code comments explaining this.
Please either remove this URL, or explain and justify why this endpoint should be in this PR!

Reply:
The register endpoint is the POST new user endpoint that does not have JWT, should I name it something more clear?

Reply:
Yes please!

Reply:
could also be `api/users`register`